test/Common/TestHelpers.cs

```diff
@@ -89,9 +89,15 @@ internal static class TestHelpers
                 testNameResolver);
         }
 
+        // Create a valid task hub from the test name, and add a random g
```

Comment:
I think this comment might be incomplete / cut-off?

Reply:
Haha probably was debating between leaving the comment or not and my brain just gave up :)